src/EditorFeatures/Test2/Rename/RenameEngineTests.CSharpConflicts.vb

```diff
@@ -3604,6 +3604,28 @@ class C
 {
     public int MyProperty { get; {|conflict:set|}; }
 
+    private void {|conflict:$$_set_MyProperty|}(int value) => throw null;
+}
+                            </Document>
+                        </Project>
+                    </Workspace>, host:=host, renameTo:="set_MyProperty")
+
+                result.AssertLabeledSpansAre("conflict", type:=RelatedLocationType.UnresolvedConflict)
+            End Using
+        End Sub
+
+        <WpfTheory>
+        <WorkItem(45677, "https://github.com/dotnet/roslyn/issues/45677")>
+        <CombinatorialData, Trait(Traits.Feature, Traits.Features.Rename)>
+        Public Sub ConflictWhenRenamingPropertyInitterLikeMethod(host As RenameTestHost)
+            Using result = RenameEngineResult.Create(_outputHelper,
+                    <Workspace>
+                        <Project Language="C#" CommonReferences="true">
+                            <Document>
+class C
+{
+    public int MyProperty { get; {|conflict:init|}; }
+
     private void {|conflict:$$_set_MyProperty|}(int value) => throw null;
```

Comment:
Also adding @davidwengier as he worked on this a while ago.
The `init` accessor is compiled in the IL to `set_MethodName`.


```csharp
public class C {
    public int X {get; init;}
}
```

```il
    .property instance int32 X()
    {
        .get instance int32 C::get_X()
        .set instance void modreq(System.Runtime.CompilerServices.IsExternalInit) C::set_X(int32)
    }
```

@davidwengier Can you confirm if the added test is correct?

Reply:
Yes, the change suggested in your comment looks right to me.